server/models/update/schema-v2.sql

```diff
@@ -0,0 +1,125 @@
+--
```

Comment:
The schema is very important.  The file, not so much.   `synt.sql` looked like it mostly contained data changes, so I made an upgrade SQL for the schema.  That way, we'll know what schema changes when introduced the when we try to do a 1.X --> 2.X upgrade.

This already contains the upgrade script to migrate data in the 1.X cash table (the one found at our test hospital) to the 2.X cash table.